src/http/api/note.ts

```diff
@@ -35,17 +36,21 @@ export async function handleCreateNote(
 
     if (data.imageUrl) {
         imageUrl = new URL(data.imageUrl);
-    } else if (data.image?.url) {
+    } else if (data.image) {
         imageUrl = new URL(data.image.url);
     }
 
-    const altText = data.image?.altText ?? undefined;
+    const image: ImageAttachment | undefined = imageUrl
```

Comment:
This will be cleaned up after UI changes, we don't have to construct this here we can directly pass the image object received in the req params.